lib/Controller.php

```diff
@@ -539,16 +542,10 @@ private function _jsonld($type)
      * Proxies a link using the specified proxy class, and updates the status or error with the response.
      *
      * @access private
-     * @param string $link The link to be proxied.
-     * @param string $proxyClass The fully qualified class name of the proxy to use.
+     * @param AbstractProxy $proxy The instance of the proxy class.
      */
-    private function _shortenerproxy($link, $proxyClass)
+    private function _shortenerproxy(AbstractProxy $proxy)
     {
-        if (!is_subclass_of($proxyClass, AbstractProxy::class)) {
```

Comment:
Hmm why did you remove this safety check? IMHO guarding input arguments like this is best practice/defensive coding and fine…

Ah because `AbstractProxy` in PHP already does this? Then that's fine.